microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/MultipleJarDeploymentTest.java

```diff
@@ -65,30 +59,9 @@
                 .addAsManifestResource(new StringAsset(mpConfig), "microprofile-config.properties");
     }
 
-    /**
-     * @tpTestDetails Deploy first and then second MP FT application. Both of them are the same (same classes/methods)
-     * @tpPassCrit Verify that second deployment causes that there is logged warning: "WFLYMPFTEXT0002: Hystrix was already
-     *             configured!
-     *             ..." and there is no ERROR message.
-     * @tpSince EAP 7.4.0.CD19
-     */
-    @Test
-    @RunAsClient
-    public void testDeploySecondJarLogsHystrixNoConfigChangeWarning() throws Exception {
-        try (final OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
-            final LogChecker logChecker = new ModelNodeLogChecker(client, 10, true);
-
-            Assert.assertTrue(
-                    "Log does not contain WARNING Log: WFLYMPFTEXT0002: Hystrix was already configured! Skipping configuration from deployment - but it should.",
-                    logChecker.logMatches(Pattern.compile(".*WFLYMPFTEXT0002.*")));
-            Assert.assertFalse("Log must not contain any ERROR logs but there are. See server log." + logChecker.toString(),
-                    logChecker.logMatches(Pattern.compile(".*ERROR.*")));
-        }
```

Comment:
Even though this is not supported, there could be checked that no warning was logged with 2nd deployment. Or perhaps that multiple deployments are not supported?

Reply:
As far as I know, multiple deployments are not supported, but no warning is being logged about this so there is nothing to test. I also don't want to test that code `WFLYMPFTEXT0002` is not present because it was removed and could be used for some other warning in the future.